docs/user-guide/global-configurations/deployment-charts.md

```diff
@@ -1,70 +1,94 @@
+| `Name` | Name of the Helm chart (Required). |
 | `Version` | This is the chart version. Update this value for each new version of the chart (Required). |
-| `Description` | Description of the chart (Optional). |
+| `Description` | Give a description to your chart (Optional). |
+
+{% hint style="info" %}
+### Example of Chart.yaml
+[Click here](https://devtron-public-asset.s3.us-east-2.amazonaws.com/custom-charts/chart-yaml-file.png) to view a sample 'Chart.yaml' file.
+{% endhint %}
 
-Please see the following example:
+### 2. Create an Image Descriptor Template File
 
-![Chart.yaml file](https://devtron-public-asset.s3.us-east-2.amazonaws.com/custom-charts/chart-yaml-file.png)
+The Image Descriptor Template file is a GO template that produces a valid JSON file upon processing. It allows Devtron to dynamically inject values from the CD pipeline into your Helm chart during deployment. Therefore, details like image repository, tag, and environment are automatically populated at the placeholders specified in `.image_descriptor_template.json`.
 
-### 2. Create the image descriptor template file `.image_descriptor_template.json`
+* In the root directory of your chart, create a file named `.image_descriptor_template.json` using the following command:
 
-It's a GO template file that should produce a valid `JSON` file upon rendering. This file is passed as the last argument in
-`helm install -f myvalues.yaml -f override.yaml` command.
+    ```bash
+    touch .image_descriptor_template.json
+    ```
 
-Place the `.image_descriptor_template.json` file in the root directory of your chart.
+* Ensure the above file is created in the directory where the main `Chart.yaml` exists (as shown below): 
 
-You can use the following variables in the helm template (all the placeholders are optional):
+    ![Figure 2: Filepath of Image Descriptor Template](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-charts/descriptor-filepath.jpg)
 
-> The values from the CD deployment pipeline are injected at the placeholder specified in the `.image_descriptor_template.json` template file.
+* Paste the following content in `.image_descriptor_template.json` file:
 
-```bash
-{
-    "server": {
-        "deployment": {
-            "image_tag": "{{.Tag}}",
-            "image": "{{.Name}}"
+    ```bash
+    {
+        "server": {
+            "deployment": {
+                "image_tag": "{{.Tag}}",
+                "image": "{{.Name}}"
+            }
+        },
+        "pipelineName": "{{.PipelineName}}",
+        "releaseVersion": "{{.ReleaseVersion}}",
+        "deploymentType": "{{.DeploymentType}}",
+        "app": "{{.App}}",
+        "env": "{{.Env}}",
+        "appMetrics": {{.AppMetrics}}
+    }
+    ```
+
+    You can customize this template to include only the values your deployment needs. For instance, if you only require the image repository and tag, your template would look like:
```

Comment:
We need to elaborate a bit on how the image descriptor is used, what changes is the user suppose to do our how will they get the values in their chart / values.yaml